nixos/lib/test-driver/test_driver/__init__.py

```diff
@@ -112,6 +134,15 @@ def main() -> None:
 
     args = arg_parser.parse_args()
 
+    # print the info needed to update another Driver
+    # TODO: properly escape these arguments
```

Comment:
Is this TODO important?

Reply:
The only cases I can think of where this would cause a problem are
1. if someone had a truly psychotic CWD with a newline in it.
2. if they're doing something very weird with the test driver by passing arguments in manually **and** they're using paths with spaces (or newlines) in them.

Technically not impossible, but I would be surprised if it came up in practice. And escaping stuff means decoding it on the other end. It just adds a bunch of complexity so I didn't bother.

Reply:
I guess it works then, yeah.

If you could send a JSON instead of a custom stream, that could also simplify the reading code, and make those parts of the program easier to correlate (i.e. when browsing with `grep` or similar)